pkg/server/virtual.go

```diff
@@ -84,6 +91,18 @@ func newVirtualConfig(
 		return nil, err
 	}
 
+	// apply admission to check permission claims for resources through a view's URL
+	admissionPluginInitializers := []admission.PluginInitializer{
+		kcpadmissioninitializers.NewKcpInformersInitializer(kcpSharedInformerFactory, cacheKcpSharedInformerFactory),
+	}
+	o.Virtual.VirtualWorkspaces.Admission.ApplyTo(
+		&recommendedConfig.Config,
+		informerfactoryhack.Wrap(kubeSharedInformerFactory),
+		clientsethack.Wrap(kubeClusterClient),
+		utilfeature.DefaultFeatureGate,
+		admissionPluginInitializers...,
+	)
+
```

Comment:
Shouldn't this go to `virtualrootapiserver.NewConfig`? It's strange to have this duplicated in the standalone command.

Reply:
or even in the apiexport specific apiserver.

Reply:
We weren't sure the best place to put it. Thanks for the suggestions!